### PR TITLE
EasySolved

### PR DESCRIPTION
**Pull request by JethroGibbsN**
Check 97 to 125 line numbers and use it where ever you want

### CODE REVIEW

File: Server/controllers/participationTut.controller.js [hunk 1]
```diff
@@ -93,6 +93,34 @@ exports.acceptSubmission = (sub, callback) => {
                             }
                             return callback("Error updating Participation with Id ", null);
                         });
+
+                       if(sub.difficulty === 'Easy'){
+                        if (participation.EasySolved.length !== 0){
+                            for(let k = 0; k < participation.EasySolved.length; k++){
+                                if (participation.EasySolved[k].questionId !== sub.questionId){
+                                        Participation.findOneAndUpdate({participationId: sub.participationId}, {$addToSet:{
+                                            EasySolved: { questionId: sub.questionId, difficulty: sub.difficulty}
```
Comment (flozender) [inline Server/controllers/participationTut.controller.js]: Don't need to add an object here, just push the QuesID as a string, and you're not even checking if it already exists, do that.

File: Server/index2.js [hunk 2]
```diff
@@ -0,0 +1,330 @@
+const express = require('express');
```
Comment (flozender) [inline Server/index2.js]: Why do we have index2 again?

File: Server/practice.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+arr = [{qid: 'abc'}, {qid: 'bcd'}]
```
Comment (flozender) [inline Server/practice.js]: What is this?

File: Server/index2.js (hunk 2 above)
Comment (JethroGibbsN) [reply Server/index2.js]: No not required

File: Server/practice.js (hunk 3 above)
Comment (JethroGibbsN) [reply Server/practice.js]: Just for me, Ill delete

File: Server/index2.js (hunk 2 above)
Comment (flozender) [reply Server/index2.js]: Why did you add it then, remove it

File: Server/controllers/participationTut.controller.js [hunk 4]
```diff
@@ -121,6 +123,24 @@ exports.acceptSubmission = (sub, callback) => {
...
+                //             participation.EasySolved.push
+                //         }
+                //     }
+                //  }
+                let compare = inarray(participation.EasySolved, sub.questionId);
+                if (!compare){
+                    participation.EasySolved.push(sub.questionId);
+                }
+                } else{
+                    participation.EasySolved.push(sub.questionId);
```
Comment (flozender) [inline Server/controllers/participationTut.controller.js]: You're adding it in both conditions? What are you doing here?

File: Server/practice.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+a = 20
```
Comment (flozender) [inline Server/practice.js]: What is this??

File: Server/controllers/participationTut.controller.js (hunk 4 above)
Comment (JethroGibbsN) [reply Server/controllers/participationTut.controller.js]: In first condition I am checking if the length is not 0. if it is then i am comparing the  question ids with the question id in the result, if it is not matching then i am pushing. Or else if the length of array is 0, no need to check, so directly push. thats what I did

File: Server/practice.js (hunk 5 above)
Comment (JethroGibbsN) [reply Server/practice.js]: This will not be there I forgot to delete. NOw I will delete. This is not related to app

File: Server/index2.js (hunk 2 above)
Comment (JethroGibbsN) [reply Server/index2.js]: okay

**Pull request merged**